server/routes/contacts/add/domestic-status/addContactDomesticStatusController.test.ts

```diff
@@ -218,6 +255,22 @@ describe('POST /prisoner/:prisonerNumber/contacts/create/domestic-status/:journe
   })
 
   it('should return to start if no journey in session', async () => {
+    app = appWithAllRoutes({
+      services: {
+        auditService,
+        prisonerSearchService,
+        referenceDataService,
+        contactsService,
+        alertsService,
+      },
+      userSupplier: () => currentUser,
+      sessionReceiver: (receivedSession: Partial<SessionData>) => {
+        session = receivedSession
+        session.addContactJourneys = {}
+        session.addContactJourneys[journeyId] = { ...existingJourney }
+      },
+    })
```

Comment:
Question - What is the need for the addition of the appWithAllRoutes() in quite a few tests What changed to make the tests require this?